src/spikeinterface/preprocessing/tests/test_decimate.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from pathlib import Path
+
+import itertools
+from spikeinterface import NumpyRecording
+from spikeinterface.core import generate_recording
+from spikeinterface.preprocessing.decimate import DecimateRecording
+import numpy as np
+
+
+def test_decimate():
+    rec = generate_recording()
+
+    N = 101
+    rec = NumpyRecording([np.arange(N).reshape(N,1)], 1)
+    parent_traces = rec.get_traces()
+
+    for decimation_frame_start, decimation_frame_step in itertools.product(
+        [0, 1, 9, 10, 11, 200],
+        [1, 9, 10, 11, 200]
```

Comment:
It would be great to use `@pytest.mark.parametrize` to expand tests, see for example [here](https://github.com/SpikeInterface/spikeinterface/blob/main/src/spikeinterface/preprocessing/tests/test_interpolate_bad_channels.py#L84-L88)

Reply:
got lazy sorry